beacon-chain/p2p/service.go

```diff
@@ -533,6 +536,15 @@ func (s *Service) addKadDHTNodesToExclusionList(addr string) error {
 	return nil
 }
 
+func (s *Service) updateBitfield(bitV bitfield.Bitvector64) {
```

Comment:
should this be next to `retrieveIndicesFromBitfield` where `status.go` is?  or `subnets.go`